test/utils/Utils.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.23;
+
+import { Test, console2 } from "forge-std/Test.sol";
+import { Utils } from "script/contracts/Utils.sol";
+
+contract UtilsTest is Test {
+    Utils utils;
+
+    function setUp() public {
+        vm.setEnv("NETWORK", "testnet");
```

Comment:
`testnet` hardcoded word is also used inside some tests like `test_getL1AddressesFilePath`.
Maybe add it as a string global variable and use inside all tests where applicable.